### PR TITLE
Rename the SuggestedGenes schema name to be singular

### PR DESCRIPTION
**Pull request by xtina-starr**
Simple name changes to follow convention.

### CODE REVIEW

File: schema/suggested_genes.js [hunk 1]
```diff
@@ -12,10 +12,10 @@ const SuggestedGeneType = new GraphQLObjectType({
 })
 
 const SUGGESTED_GENES_JSON = "https://s3.amazonaws.com/eigen-production/json/eigen_categories.json"
-const SuggestedGenes = {
+const SuggestedGene = {
   type: new GraphQLList(SuggestedGeneType),
-  description: "List of curated genes wit custom images",
+  description: "List of curated genes with custom images",
```
Comment (craigspaeth) [inline schema/suggested_genes.js]: Seems odd to me that a "List of curated genes" would be singular "SuggestedGene", but I assume there's a good reason for that?

**Comment by craigspaeth** [comment]
Wanted to point out the odd List == Singular naming, but code LGTM, feel free to merge if you are aware of that and I'm just missing context 👍

File: schema/suggested_genes.js (hunk 1 above)
Comment (orta) [reply schema/suggested_genes.js]: yeah,  the terminology is a bit weird

```js
const SUGGESTED_GENES_JSON = "https://s3.amazonaws.com/eigen-production/json/eigen_categories.json"
const SuggestedGene = {
  type: new GraphQLList(SuggestedGeneType),
  description: "List of curated genes with custom images",
  resolve: () => fetch(SUGGESTED_GENES_JSON).then(({ body }) => body),
}
```

this should be `const SuggestedGenes = {` as that is the definition of the function. The type of `SuggestedGene`  is fine

```
const SuggestedGeneType = new GraphQLObjectType({
  name: "SuggestedGene",
  fields: {
    id: { type: GraphQLString },
    image_url: { type: GraphQLString },
    _id: { type: GraphQLString },
    name: { type: GraphQLString },
  },
})
```

File: schema/suggested_genes.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import fetch from "../lib/apis/fetch"
 import { GraphQLObjectType, GraphQLList, GraphQLString } from "graphql"
 
 const SuggestedGeneType = new GraphQLObjectType({
-  name: "SuggestedGenes",
+  name: "SuggestedGene",
```
Comment (xtina-starr) [inline schema/suggested_genes.js]: @orta a bit confused. Is the name on `SuggestedGeneType` the only thing that needs to change?

**Comment by xtina-starr** [comment]
Fixed.

**Pull request merged**